importer/importer.go

```diff
@@ -428,7 +428,16 @@ func ExtractEntry(entryElement xml.Node, oid string, extractor EntryExtractor, s
 	// create status code and set status code from status
 	set_status_code(&entry, status)
 
-	fullEntry := extractor(&entry, entryElement)
+	var fullEntry interface{}
+
+	//if there is no entry extractor method provided, generically extract code
+	if extractor == nil {
+		codeXPath := xpath.Compile("cda:code")
+		ExtractCodes(&entry.Coded, entryElement, codeXPath)
+		fullEntry = entry
+	} else {
+		fullEntry = extractor(&entry, entryElement)
+	}
```

Comment:
I don't know if this was a style decision, but if not, 431-440 could be condensed slightly to 
```go
if extractor == nil {
	ExtractCodes(&entry.Coded, entryElement, xpath.Compile("cda:code"))
	return entry
}
return extractor(&entry, entryElement)
```